pkg/metrics/metrics.go

```diff
@@ -71,41 +74,69 @@ func NewServerMetrics() *ServerMetrics {
 				},
 				[]string{scheduleLabel},
 			),
+			backupDurationSeconds: prometheus.NewHistogramVec(
+				prometheus.HistogramOpts{
+					Namespace: metricNamespace,
+					Name:      backupDurationSeconds,
+					Help:      "Time taken to complete backup, in seconds",
+					// Split backup observations into buckets based on number of minutes.
+					Buckets: []float64{
+						1 * secondsInMinute,
+						2 * secondsInMinute,
+						3 * secondsInMinute,
+						4 * secondsInMinute,
+						5 * secondsInMinute,
+						6 * secondsInMinute,
+						7 * secondsInMinute,
+						8 * secondsInMinute,
+						9 * secondsInMinute,
+						10 * secondsInMinute,
```

Comment:
This will mean any backup longer than 10 minutes is counted as "infinite". Is that reasonable?

It might be useful to use [exponential buckets](https://godoc.org/github.com/prometheus/client_golang/prometheus#ExponentialBuckets) to get more resolution at the higher end of this range.